apps/src/code-studio/components/Libraries/LibraryManagerDialog.jsx

```diff
@@ -46,37 +95,73 @@ export default class LibraryManagerDialog extends React.Component {
     );
   };
 
-  displayLibraries = () => {
-    return this.state.libraries.map(library => {
+  displayProjectLibraries = () => {
+    let libraries = this.state.libraries;
+    if (!Array.isArray(libraries) || !libraries.length) {
```

Comment:
In what case would `libraries` not be an array?

Reply:
libraries could be `undefined`. `!libraries` would achieve the same thing. I just used isArray to be more specific.